.travis.yml

```diff
@@ -17,53 +18,36 @@ env:
     - ES_JAVA_OPTS="-Xms256m -Xmx512m"
 
   matrix:
-    - MODULE='berkeleyje'
-    - MODULE='cassandra'
-    - MODULE='es' ARGS='-DthreadCount=1'
-    - MODULE='es' ARGS='-DthreadCount=1 -Pelasticsearch2'
     - MODULE='hadoop-parent/janusgraph-hadoop-2'
-    - MODULE='hbase-parent/janusgraph-hbase-098'
-    - MODULE='hbase-parent/janusgraph-hbase-10'
-    - MODULE='lucene' ARGS='-DthreadCount=1'
-    - MODULE='solr' ARGS='-DthreadCount=1'
+    - MODULE='lucene'
```

Comment:
Interesting, thanks for clarifying and that's a good idea! Do you think we should add a comment to that effect for future editors of the file so they know why it's ordered the way it is?